src/PrestaShopBundle/Form/Admin/Sell/Order/Invoices/InvoiceOptionsType.php

```diff
@@ -78,36 +80,164 @@ public function __construct(
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $featureTranslationDomain = 'Admin.Orderscustomers.Feature';
```

Comment:
@PrestaShop/prestashop-maintainers 

The previous mechanism was scanning files. How does the Symfony translator export a catalog of translations? Will it work?

if so... wouldn't it be good to have a class with translation domain constants? 🤔

Reply:
Sorry @kpodemski I am not sure I understand your point. Is your comment related to @jolelievre comment ?